modules/data/pages/define-and-deploy-the-bdm.adoc

```diff
@@ -252,35 +252,87 @@ To modify a new or existing object:
+The `version` and `groupId` of those artifacts matches the Bonita project version. 
+You have the possibility to view and retrieve the `bdm-model` dependency from the BDM editor `Maven artifact properties` section. +
 Those maven artifacts are meant to be used from REST API extensions, using the following dependencies:
 
+.The generated model dependency template
 [source,xml]
 ----
 <dependency>
-	<groupId>[YOUR GROUP ID]</groupId>
-	<artifactId>bdm-client</artifactId>
-	<version>1.0.0</version>
-	<scope>provided</scope>
+	<groupId>[PROJECT GROUP ID]</groupId>
+	<artifactId>[PROJECT ID]-bdm-model</artifactId>
+	<version>[PROJECT VERSION]</version>
+	<scope>provided</scope> <1>
 </dependency>
+----
+<1> In most use cases, when the dependent project is meant to be deployed server side (eg: Rest API Extension, Custom libraries used in process expressions...etc.) the dependency scope of the model should be `provided`. When used and packaged as a client application, `compile` scope (maven default) should be used.
+
+Contains all Business Object entities and DAO interfaces.
+Can be used as dependency of:
 
+* The main application module
+* Any Rest API Extension
+* A Java client application using the https://search.maven.org/search?q=g:org.bonitasoft.web%20AND%20a:bonita-java-client[bonita-java-client^] and the https://github.com/bonitasoft/bonita-java-client#41-bdm-typed-queries[Bdm typed queries^]
+* Integration tests, using the xref:test-toolkit::business-data.adoc#_as_a_typed_object[Bonita Test Toolkit^] and the typed Business Objects.
+
+.The generated dao-client dependency template
+[source,xml]
+----
 <dependency>
-	<groupId>[YOUR GROUP ID]</groupId>
-	<artifactId>bdm-dao</artifactId>
-	<version>1.0.0</version>
-	<scope>provided</scope>
+    <groupId>[PROJECT GROUP ID]</groupId>
+    <artifactId>[PROJECT ID]-bdm-dao-client</artifactId>
+    <version>[PROJECT VERSION]</version>
 </dependency>
 ----
+Contains client side implementation of the Business Objects DAO.
+Can be used as dependency of a java client application using the https://search.maven.org/search?q=g:org.bonitasoft.engine%20AND%20a:bonita-client[bonita-client^] and the https://javadoc.bonitasoft.com/api/{javadocVersion}/org/bonitasoft/engine/api/APIClient.html#getDAO(java.lang.Class)[APIClient#getDAO^] factory method.
+
+The `bdm-model` dependency is automatically added when a REST API Extension is created from the Bonita Studio. It allows to manipulate Business Objects from a REST API Extension. +
+
+== Build and Deploy BDM artifacts
+
+It is possible to automate the build and deployment of the `bdm-model` and `bdm-dao-client` artifacts using *only* Maven. Publishing those artifacts can have its purpose if you need to consume them in any other maven or gradle projects outside of a Bonita project.
+
+[CAUTION]
+It is not supported for a Bonita project to depend on the `bdm-model` dependency from another Bonita project.
```

Comment:
```suggestion
It is not supported for a Bonita project to depend on the `bdm-model` from another Bonita project.
```
Other question on my side: "is it a valide use case to have a rest api in my project1 that need to do CRUD using  project1-bdm-model and also depends on project2-bdm-model just to know the object structure?"
If yes, we should be more explicit for this "CAUTION", to avoid the confusion between the current project BDM used to allows CRUD for the project, and other BDM dependencies?